homework6/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.3"
+services:
+  db:
+    image: linuxserver/mariadb:latest
+    container_name: db
+    restart: always
+    environment:
+      MYSQL_USER: root
+      MYSQL_PASSWORD: password
+      MYSQL_ROOT_PASSWORD: password
+  baseball:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: baseball
+    volumes:
+      - ./results:/results
+    restart: unless-stopped
```

Comment:
two containers created well.
volume linked to a local folder is a good idea to get the results.